website/server/controllers/api-v3/worldState.js

```diff
@@ -0,0 +1,92 @@
+import common from '../../../common';
+import _ from 'lodash';
+import {
+  model as Group,
+  TAVERN_ID as tavernId,
+} from '../../models/group';
+import {
+  getCachedResponse,
+  saveResponseToDisk,
+} from '../../libs/cachedResponse';
+import { langCodes } from '../../libs/i18n';
+
+const RESPONSE_NAME = 'world_state';
+
+let api = {};
+
+function getSpecialItems () {
+  return _(common.content.hatchingPotions)
```

Comment:
Heads up, we're migrating to lodash v4 (https://github.com/HabitRPG/habitica/pull/8495) which includes many breaking changes (https://github.com/lodash/lodash/wiki/Changelog#v400). Before merging this PR should be checked and updated to make sure all lodash methods are compatible with v4.